dissect/target/plugins/os/unix/log/messages.py

```diff
@@ -40,34 +61,33 @@ def messages(self):
         Sources:
             - https://geek-university.com/linux/var-log-messages-file/
         """
-        for f in self.target.fs.glob_ext("/var/log/messages*"):
-            # FIXME Year is not part of syslog timestamp, take it from file. does not work yet for tar.
-            # FIXME watch for change of year in file!!!
-            # year = datetime.datetime.fromtimestamp(f.stat.mtime).year
-            year = 2018
-            fh = f.open()
-            if f.name[-2:] == "gz":
-                fh = zlib.decompress(fh.read(), 31)
-                try:
-                    fh = StringIO(fh.decode())
-                except UnicodeDecodeError:
-                    pass
-            if "bz2" in f.name:
-                fh = bz2.decompress(fh)
-                fh = StringIO(fh.decode())
-
-            for line in fh:
+        syslogs = self.target.fs.path("/var/log/").glob("syslog*")
+        messages = self.target.fs.path("/var/log/").glob("messages*")
+        log_files: Iterator[TargetPath] = chain(syslogs, messages)
+
+        for log_file in log_files:
+
```

Comment:
```suggestion
        var_log = self.target.fs.path("/var/log")
        for log_file in chain(var_log.glob("syslog*"), var_log.glob("messages*"):
```
Might be a bit cleaner.